### DISCUSSION TITLE
Refresh Token Expires in 24 hours taken by oauth playground

### DISCUSSION THREAD

**Discussion by humzakhaled** [masbug/flysystem-google-drive-ext#93] [General]
When we get our refresh token from [https://developers.google.com/oauthplayground](https://developers.google.com/oauthplayground ) it revokes after 24 hours. They also mentioned a note on step 3 
`Note: The OAuth Playground will automatically revoke refresh tokens after 24h. You can avoid this by specifying your own application OAuth credentials using the Configuration panel.`

I was using nao-pan package before with Laravel and i was facing `Daily limit usage expired`. Now I have installed `masbug/flysystem-google-drive-ext`. May I know if this package handles this error and refreshes the token? How can i make refresh token forever so it daily limit usage doesn't exceed? Please suggest me something permanent for this issue. Im stuck.

**Comment by parallels999** [comment]
> May I know if this package handles this error and refreshes the token?

Yes, https://github.com/masbug/flysystem-google-drive-ext/pull/35

>How can i make refresh token forever so it daily limit usage doesn't exceed?

The package handle it

**Reply by humzakhaled** [reply]
Thanks for the reply. I installed this fresh package two days ago. So, I don't have to change anything. right?

**Reply by parallels999** [reply]
> So, I don't have to change anything. right?

Just follow documentation
